platformproxy/VncProxyPlugin.cpp

```diff
@@ -17,7 +17,7 @@ namespace
         {
             QPlatformIntegration* integration = nullptr;
 
-            if ( system.startsWith( "vnc", Qt::CaseInsensitive ) )
+            if ( system.startsWith( QStringLiteral("vnc"), Qt::CaseInsensitive ) )
```

Comment:
Could you please isolate the QT_NO_CAST_FROM_ASCII related fixes into an extra pull request ?

Reply:
Sure, I should have done that from the start.